src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -54,7 +54,7 @@ public static void CheckFileOutOfSourceControl(string file)
 
         public static void AddFileToProject(this Project project, string file, string itemType = null)
         {
-            if (project.IsKind(ProjectTypes.ASPNET_5, ProjectTypes.DOTNET_Core, ProjectTypes.SSDT))
+            if (IsCapabilityMatch(project, Constants.DotNetCoreWebCapability))
```

Comment:
Wait, is this right? We are *not* going to add libman.json to .Net Core projects? I think we were excluding only whatever SSDT subtype was. Now we are excluding *all* .Net Core web projects, right? I think we need to double-check with Mads on what he was trying to exclude. I think after this change you won't be able to do "Manage Client Libaries" (which should libman.json) in all projects.

Reply:
Nevermind, I'm being slow, .net core adds everything to the project automatically, so it's a no-op for .net core.